openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -550,11 +551,24 @@ port_INLINE void activity_synchronize_endOfFrame(PORT_RADIOTIMER_WIDTH capturedT
       ieee154e_vars.dataReceived->l2_frameType = ieee802514_header.frameType;
       ieee154e_vars.dataReceived->l2_dsn       = ieee802514_header.dsn;
       memcpy(&(ieee154e_vars.dataReceived->l2_nextORpreviousHop),&(ieee802514_header.src),sizeof(open_addr_t));
-      
-      // toss the IEEE802.15.4 header -- this does not include IEs as they are processed 
+
+      // This is the case where we first hear an EB and do not yet know the ASN used

```

Comment:
for some reason, I get a notification of this comment by mail, but don't
see it in https://github.com/openwsn-berkeley/openwsn-fw/pull/167.
@malisav, can you confirm all suggestions are addressed, or just the one
about ASN?

On Thu, Jun 18, 2015 at 3:11 PM, Mališa Vučinić <notifications@github.com>
wrote:

> In openstack/02a-MAClow/IEEE802154E.c
> <https://github.com/openwsn-berkeley/openwsn-fw/pull/167#discussion_r32727270>
> :
>
> > @@ -550,11 +551,24 @@ port_INLINE void activity_synchronize_endOfFrame(PORT_RADIOTIMER_WIDTH capturedT
> >        ieee154e_vars.dataReceived->l2_frameType = ieee802514_header.frameType;
> >        ieee154e_vars.dataReceived->l2_dsn       = ieee802514_header.dsn;
> >        memcpy(&(ieee154e_vars.dataReceived->l2_nextORpreviousHop),&(ieee802514_header.src),sizeof(open_addr_t));
> > -
> > -      // toss the IEEE802.15.4 header -- this does not include IEs as they are processed
> > +
> > +      // This is the case where we first hear an EB and do not yet know the ASN used
>
> @twatteyne <https://github.com/twatteyne> can you take a look if the
> latest commits address what you had in mind?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openwsn-berkeley/openwsn-fw/pull/167/files#r32727270>.
>